### PR TITLE
Po badge

### PR DESCRIPTION
**Pull request by ritikaawasthy**

### CODE REVIEW

**Review by Ananyamadhu08 — APPROVED** [review]
looks good to go, good UI 👍

File: index.html [hunk 1]
```diff
@@ -1,38 +1,249 @@
...
+        <li>Modal</li>
+        <li>Grid</li>
+        <li>Utility</li>
+      </ul>
+    </section>
+    <section class="layout-main">
+      <div class="container-col">
+
+        <h2>ALERT</h2>
+        <p class='f-s'>Alert components using Tailwind CSS to speed up your project.</p>
```
Comment (nisargpawade) [inline index.html:38]: change the description of alert "tailwind css".

**Review by nisargpawade — COMMENTED** [review]
everything else looks good

**Review by nisargpawade — APPROVED** [review]

**Comment by ashishkatiyar2404** [comment]
comments some change.
your UI looks amazing💯
your code looks good to merge.

**Pull request merged**